Library/Homebrew/tab.rb

```diff
@@ -12,10 +12,11 @@ class Tab
   extend Cachable
 
   FILENAME = "INSTALL_RECEIPT.json"
+  SPDX_FILENAME = "spdx.sbom.json"
```

Comment:
Please use the official naming conventions of LF/SPDX: `*.spdx.json` as according to official guidelines https://spdx.dev/learn/overview/
> [...] use one of five file formats [...] JSON(`.spdx.json`) [...].

This will help proper detection, as well as adding other SBOM formats, like CycloneDX.

This acknowledged by LF/OpenSSF other guidelines https://github.com/ossf/sbom-everywhere/blob/main/reference/sbom_naming.md